workbench-core/swb-common-ui/src/api/auth.ts

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 /*
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
 
 import { httpApiGet, httpApiPost } from './apiHelper';
 
+// eslint-disable-next-line @rushstack/no-new-null
```

Comment:
Is `null` preferable to `undefined` here? The [rush eslint rule](https://www.npmjs.com/package/@rushstack/eslint-plugin) makes the case that `null` should be avoided if possible

Reply:
This is intentional. `null` is required here since that's what's expected from the caller